### DISCUSSION TITLE
Latest project pipeline

### DISCUSSION THREAD

**Discussion by jeremad** [python-gitlab/python-gitlab#2696] [Ideas]
Hello,
Not sure if this is the right place, but is there any plan to support the `latest` API for pipelines ?
https://docs.gitlab.com/ee/api/pipelines.html#get-the-latest-pipeline

Cheers

**Comment by nejch** [comment]
@jeremad this should already be possible with something like this:

```
pipeline = project.pipelines.get("latest")
```

**Reply by jeremad** [reply]
And would it work with `pipeline = project.pipelines.get("latest?ref=my_branch")` ?

**Reply by nejch** [reply]
The proper way to do this in python would be:

```python
pipeline = project.pipelines.get("latest", ref="my_branch")
```

This should work, yes!

**Reply by jeremad** [reply]
Amazed that this actually works. That's why I wanted to try a more a hacky way. Thanks for your answsers. A small mention in the doc would be amazing though. Cheers.

**Reply by nejch** [reply]
@jeremad that's great, if you feel like it you can open a PR in https://github.com/python-gitlab/python-gitlab/blob/main/docs/gl_objects/pipelines_and_jobs.rst!
